rag/svr/task_executor_refactor/dataset_skill_generator.py

```
@@ -220,22 +221,57 @@ async def doc_summary_for_skill(
     )
 
 
-def build_skill_md(node: "SkillNode") -> str:
+_SENTENCE_ENDINGS = "。！？!?；;"
```

Ruff (0.16.1)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)